[WEBP-4987] Do/Don't Block + Docs update (#3)
victor-move/exporter-rdc-documentation: documentation/BLOCK_CREATION.md

> A step by step guide on creating custom blocks and any "gotchas" you may need to worry about.

Reviewer: ![image](https://github.com/victor-move/exporter-rdc-documentation/assets/101747726/2957895d-cba9-4109-ada6-a9e3e4fb291c)
if they are varying height, are they suppose to be the same height

Reviewer: and it looks squished if we put 3 here, kel's figma is at 1200px not 756 in that text..

Author: Good question, let me give that a try

Author: Well actually, not sure where kelly wants them aligned, I should asked about it as well

Author: <img width="825" alt="Screen Shot 2023-06-28 at 12 37 23 PM" src="https://github.com/victor-move/exporter-rdc-documentation/assets/99918698/314b9aef-2ae1-4ece-a9e5-108f0b1b7ff8">

The Pulsar language documentation isn't fully complete, but what exists should be the majority of what is needed to develop most custom blocks: https://developers.supernova.io/pulsar-language/language-features

## Handling CSS

> In the latest versions of the exporter, CSS for custom blocks is contained in their own file within `scss/blocks`. The file will need to be imported into `scss/main.scss` e.x.: `@import './blocks/questions-section';`.

Reviewer: e.ggggg
```suggestion
In the latest versions of the exporter, CSS for custom blocks is contained in their own file within `scss/blocks`. The file will need to be imported into `scss/main.scss` e.g.: `@import './blocks/questions-section';`.
```

## Workflow

> The first step should be declaring the block in `exporter.json`. This can be commited directly in `master`. From that point, the `exporter.json` changes should be pulled in on Supernova and hard refresh. Create the block in the [_custom blocks page](https://haven.preview.supernova-docs.io/latest/resources/custom-blocks-vv1BNHL1) and hit publish.

Reviewer: do we have permission to commit directly to master?

you should use the editor link instead
https://cloud.supernova.io/ws/haven/ds/10812-haven-design-systems/latest/documentation/editor/958195

Author: ```suggestion
The first step should be declaring the block in `exporter.json`. This can be done in a separate PR before any of the other work begins. Once approved and merged, the `exporter.json` changes should be pulled in on Supernova and hard refresh. Create the block in the [_custom blocks page](https://cloud.supernova.io/ws/haven/ds/10812-haven-design-systems/latest/documentation/editor/958195) and hit publish.
```
I was gonna say there's no way around committing it to master, but that's just not true lol. It's tedious but we can just have to PRs for every custom block, it's not like we're making a ton of these

> From this point, the custom block can be developed in a new branch using the [Supernova VSCode extension](https://marketplace.visualstudio.com/items?itemName=SupernovaIO.pulsar-vsc-extension). Once your account is logged in and your workspace is set to haven, a local build can be made by pressing the "run exporter" button (alternatively found in the command palette). For CSS changes, `npm run build` will need to be ran before the local build. 

Reviewer: the plugin required an api key, do we want to mention it here

Author: ```suggestion
From this point, the custom block can be developed in a new branch using the [Supernova VSCode extension](https://marketplace.visualstudio.com/items?itemName=SupernovaIO.pulsar-vsc-extension). An API key is also required, and instructions for this can be found on the extension page. Once your account is logged in and your workspace is set to haven, a local build can be made by pressing the "run exporter" button (alternatively found in the command palette). For CSS changes, `npm run build` will need to be ran before the local build. 
```
Nice catch

> Once the block's HTML and CSS has been created, a PR can be made and should be reviewed by the Web Platform team. Note that as this repo is a fork of Supernova's make sure the PR is set to target the master of `victor-move/exporter-rdc-documentation`.

Reviewer: good reminder to change target, i forgot to do that before 😅

Author: I totally didn't make the same mistake.........
